### DISCUSSION TITLE
Tell us, what is your all-time favorite book from the 20th century that left a monumental impact on you?

### DISCUSSION THREAD

**Discussion by KiyotadaMori** [NAIST-SE/AttractingContributors#1078] [2024 Attraction Contributions]
Please add your favorite book from the 20th century to the list in the README and make Pull Request .
### The Strict Rule
Share only 20th-century books
### Link
https://github.com/KiyotadaMori/AttractingContributors-20th_century_books
### How to 

1. git remote add 20-books https://github.com/KiyotadaMori/AttractingContributors-20th_century_books
2. git fetch 20-books
3. git checkout -b add-books-YOURNAME 20-books/main
4. Add your favourite books.
5. git add README.md
6. git commit -m "add 20-books"
7. git push origin add-books-YOURNAME 
8. click "Compare and pull request" and "Create Pull Reques

**Comment by Canppchk** [comment]
I add "Brave New World" by Aldous Huxley on your list.
could you help me with my project too? Thanks!! (I only need 2 left)
https://github.com/Canppchk/AttractingContributors-Ramen
